app/controllers/users_controller.rb

```diff
@@ -34,6 +34,13 @@ def index
     @users = @roster.users
     @other_users = User.all - @users
     @fallback = @roster.fallback_user
+    @active = User.active.order :last_name
+  end
+
+  def inactive
```

Comment:
I would recommend a parameter to the `index` action that controls whether inactive users are shown, rather than a separate controller action and view.

This way, if we would like to make changes to the users index in the future, we don't need to remember to mirror those changes on the 'inactive' pages.

Additionally, I might recommend showing *either* active or inactive users, but not both, though I'm happy to be overruled on that.